Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
 
 3. Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
 ```
   (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  Sally is instance_of object
+  Tuesday is instance_of one-dimensional temporal region 
+  Wednesday is an instance of one-dimensional temporal region 
+  Arm is an instance of fiat object part
+  “Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+  “Sally participates in having no arms on Wednesday” is an instance of occurrent
+  “Sally participates in having at most one arm on Wednesday” is an instance of occurrent
+  Due to the vagueness of the statement above, we have narrowed the possible answers to the following two cases:
+  “Sally participates in having at least one arm on Tuesday precedes Sally participates in having no arms on Wednesday”
+  “Sally participates in having at least one arm on Tuesday precedes Sally participates in having at most one arm on Wednesday”
+
   (b) Every liver has some cell as part at all times it exists.
-  (c) John was a child, then an adult, then a senior. 
+  Search liver here: https://ontobee.org
+  We already have cell in the ontology.
+  Liver is a class in Uberon (UBERON_0002107), which is a subclass of material entity. 
+  https://ontobee.org/ontology/UBERON?iri=http://purl.obolibrary.org/obo/UBERON_0002107
+  Argument according to authority BS for the object property has_part_at_all_times: https://youtu.be/fkkWkTIxrNQ
+  Liver has_part_at_all_times Cell
+
+  (c) John was a child, then an adult, then a senior.
+  John is an instance_of object
+  childhood is an instance_of occurrent.
+  adulthood is an instance_of occurrent.
+  seniorhood is an instance_of occurrent.
+  “John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood.”
+
   (d) Goofus and Gallant are married at each point in a three year span. 
+  Goofus is an instance_of object
+  Gallant is an instance_of object
+  Marriage is an instance_of occurent
+  “Three years span 1” is an instance_of one-dimensional temporal region.
+  Notice that the original phrase doesn’t say anything about Goofus and Gallant being married to each other, and we won’t represent such a fact.
+  If zero-dimensional temporal region t1 is part_of the one-dimensional temporal region “three years span 1”, then Goofus participates in marriage at t1 and Galland participates in marriage at t1.
+
 ```
 
 4. Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
 ```
   (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+   ∃x (Tx ∧ ∃y (Hsy∧Ay)) ∧ ∃x (Wx ∧ ~∃y(Hsy∧Ay))
+   T: Tuesday
+   H: has
+   A: arm
+   W: Wednesday
+   s: Sally
+
   (b) Every liver has some cell as part at all times it exists.
+   ∀x∃y(Lx→Cy∧Pyx)
+   L: liver
+   C: cell
+   P: part of
+
```

Comment:
Good. But if we really introduce time here, it may look like: **∀x∀t(L(x) ∧ P(x,t) → ∃y(C(y) ∧ P (y, t) ∧ Has_Part (x,y,t)))** ---- "P(x,t)" means that x is present at time t